src/main/java/com/zerohunger/pdsmanagement/service/impl/StateManagementServiceImpl.java

```diff
@@ -74,6 +76,10 @@
 		if (grantOrderRes.isPresent()) {
 			OrderRequest orderChange = orderRequestRepo.findById(orderGrant.getRequestId()).get();
 			orderChange.setIsActive(false);
+			
+			Optional<Object> updatedStateAvailability = Optional.ofNullable(updateRation(orderGrant.getRequestId(),orderGrant.getGrantingStateName(), orderGrant.getQuantityGranted(),UpdateRation.STATE_AVAILABILITY ));
+			Optional<Object> updatedOrderRequest = Optional.ofNullable(updateRation(orderGrant.getRequestId(), orderGrant.getGrantingStateName(), orderGrant.getQuantityGranted(),UpdateRation.REQUEST_STATUS));
```

Comment:
where are we using "updatedStateAvailability" & "updatedOrderRequest" objects ? It is unused I guess, I believe you can directly perform those operations without using SWITCH case separately. Please try to keep codes as concise and precise as possible.